backend/app/schemas.py

```
@@ -1023,6 +1078,54 @@ class SystemPromptContext(BaseModel):
+    continuation_reason: Optional[str] = Field(
+        default=None,
+        description="Reason for continuing (tool_execution, length_limit, etc.)",
+    )
+    sandbox_id: Optional[str] = Field(
+        default=None, description="Daytona sandbox ID if using computer-use tools"
+    )
+
+    model_config = ConfigDict(from_attributes=True)
+
+
+class AgentRunDetailWithIterations(AgentRunDetail):
```

Ruff (0.14.3)

Undefined name `AgentRunDetail`

(F821)